dart_challenge/lib/unit_test_4.dart

```diff
@@ -0,0 +1,29 @@
+List<String> validatePassword(String password) {
```

Comment:
Function này chưa đạt nha em, em tìm hiểu **Regex** và làm lại nhé, chỉ cần 1 dòng để validate thôi, và print "valid" hoặc "not valid" là được